daisy/workflow/step_sub_workflow_test.go

```diff
@@ -1,169 +0,0 @@
-//  Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
So the tests in this file are important and are not replicated in any other tests, particularly the referencing of resources before and after subworkflow runs. This test as is will fail currently but we will want it once we hammer out how subworkflows and workflows pass resources to each other. 
Maybe leave the file but comment out the tests with a TODO to make them work?